test/celeritas/ext/GeantImporter.test.cc

```diff
@@ -336,7 +344,8 @@ TEST_F(FourSteelSlabsEmStandard, materials)
                                                       1.22808845964606,
                                                       1.31345289979559,
                                                       0.0209231725658313};
-    EXPECT_VEC_SOFT_EQ(expected_cutoff_energies, cutoff_energies);
+    EXPECT_VEC_NEAR(expected_cutoff_energies, cutoff_energies,
+                    geant4_is_v10() ? 1e-12 : 0.02);
```

Comment:
Nice. We have a few spots that would break with different G4 data versions (which is correlated to the G4 version, unless manually configured). Makes me think if we want to make tests compatible with more than 1 data version...